shark_turbine/kernel/wave/expansion.py

```diff
@@ -241,14 +243,23 @@ def expand_graph(
 
 
 def _expand_node(
-    node: CustomOp,
+    node: CustomOp | list[CustomOp],
     trace: CapturedTrace,
     dim_query: dict[IndexSymbol, int],
     dim_scaling: dict[IndexSymbol, int],
     node_index_setter: Callable[[CustomOp, dict[IndexSymbol, int]], None],
     context: ExpandedNodeMap,
 ) -> CustomOp:
     """Expand a single node in specific dimensions and recursively proceed to its inputs."""
```

Comment:
Let's modify the docstring to indicate what happens when a list of nodes is passed in